src/ocamlorg/lib/package.ml

```diff
@@ -162,6 +166,7 @@ module Info = struct
   let make ~package ~packages ~rev_deps opam =
     let open Lwt.Syntax in
     let+ rev_deps = mk_revdeps package packages rev_deps in
```

Comment:
Note also that you don't actually add the `latest` to the accumulation in the `mk_revdeps` function. It has: 

```ocaml
Lwt.return @@ ((name, string_of_formula formula) :: acc))
```

When it should add in the latest you calculated

```ocaml
Lwt.return @@ ((name, string_of_formula formula, latest) :: acc))
```

Reply:
That's why you said 'return' it. Noted!